bin/sync_django_ldap.py

```diff
@@ -41,10 +40,6 @@
 SYNC_TIMESTAMP_FILENAME = "/var/cache/%s.timestamp" % (NAGIOS_HEADER)
 NONROOT_DEFAULT_USER = 'apache'
 
-logger = fancylogger.getLogger()
-fancylogger.setLogLevelInfo()
```

Comment:
this is lost i guess. did we change the cron jobs to run with `--info`?

Reply:
We no longer run this in a cron.